### PR TITLE
Change some reference docs to versions that let users make comments

### PR DESCRIPTION
**Pull request by halatmit**

### CODE REVIEW

**Comment by halatmit** [comment]
please release these three files

File: appinventor/docs/reference/other/manyscreens.html [hunk 1]
```diff
@@ -402,7 +402,7 @@
 
 
   <body>
-
+<
```
Comment (jisqyv) [inline appinventor/docs/reference/other/manyscreens.html]: Note: Extra random "<" character. I'll remove it....

**Comment by halatmit** [comment]
Thanks.  Let me know when you've released this, so I can check.

File: appinventor/docs/reference/other/manyscreens.html [hunk 2]
```diff
@@ -435,15 +435,27 @@
...
+            iFrame itself is not editable, which is why we link to the Doc twice. -->
+
+       <a href="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/edit#heading=h.3jgqgjwwqxts"
+	   target="_blank">
+           Click here for a version of this page on which you can comment.</a>
+
        <iframe width="100%" height="800"
-       src="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/pub?embedded=true"></iframe>
-                    <!-- Content -->
+	       src="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/edit#heading=h.3jgqgjwwqxts">
```
Comment (jisqyv) [inline appinventor/docs/reference/other/manyscreens.html]: @halatmit I recommend strongly against using an editable version in the iframe. There is a fixed (small) limit to the number of people who can visit a document in edit mode. I would suggest going back to the published HTML version inside the iframe and keeping the editable version above it for people who want to comment.

File: appinventor/docs/reference/other/sizes.html [hunk 3]
```diff
@@ -402,7 +402,7 @@
 
 
   <body>
-
+<
```
Comment (jisqyv) [inline appinventor/docs/reference/other/sizes.html]: Another extra "<" character

File: appinventor/docs/reference/other/sizes.html [hunk 4]
```diff
@@ -434,177 +434,28 @@
...
+       <!-- Here's where we insert the link to the Google Doc that holds the page contents, so people
+            can comment and edit, together with an iframe that holds the ame doc.  Apparently the
+            iFrame itself is not editable, which is why we link to the Doc twice. -->
+
+       <a href="https://docs.google.com/document/d/1S1BBmFanf0KOD_W6HG35YEGwoAnlW7FZeq0cuMcBdZ0/edit#heading=h.3jgqgjwwqxts"
+	   target="_blank">
+           Click here for a version of this page on which you can comment.</a>
+
+       <iframe width="100%" height="800"
+	       src="https://docs.google.com/document/d/1S1BBmFanf0KOD_W6HG35YEGwoAnlW7FZeq0cuMcBdZ0/edit#heading=h.3jgqgjwwqxts">
```
Comment (jisqyv) [inline appinventor/docs/reference/other/sizes.html]: Again, use the published version in the iframe. Also note. I don't have access to this document, so I see a permission required screen inside the iframe.

**Comment by halatmit** [comment]
Try it now. I used the web versions in the iframes

**Comment by jisqyv** [comment]
LGTM, Change Merged.

**Pull request closed without merging**